src/gpo/assets/en-US/PowerToys.adml

```diff
@@ -10,6 +10,7 @@
       <string id="InstallerUpdates">Installer and Updates</string>
       <string id="PowerToysRun">PowerToys Run</string>
       <string id="AdvancedPaste">Advanced Paste</string>
+      <string id="FilePreviewers">File Previewers</string>
```

Comment:
Wondering if "File Explorer Addons" is the better name as it is the module name. And because we called it so in the gpo descriptions.